python/cuml/decomposition/tsvd.pyx

```diff
@@ -492,13 +505,17 @@ class TruncatedSVD:
         cdef uintptr_t trans_input_ptr = self._get_ctype_ptr(trans_input_data)
         cdef uintptr_t components_ptr = self.components_ptr
 
+        cdef cumlHandle* handle_ = <cumlHandle*><size_t>self.handle.getHandle()
+
         if gdf_datatype.type == np.float32:
-            tsvdTransform(<float*> input_ptr,
+            tsvdTransform(handle_[0],
+                                 <float*> input_ptr,
                                  <float*> components_ptr,
                                  <float*> trans_input_ptr,
                                  params)
         else:
-            tsvdTransform(<double*> input_ptr,
+            tsvdTransform(handle_[0],
+                                 <double*> input_ptr,
                                  <double*> components_ptr,
                                  <double*> trans_input_ptr,
                                  params)
```

Comment:
Is here a sync missing (access to `trans_input_data` in line 525.

Reply:
Similar to my above comment [here](https://github.com/rapidsai/cuml/pull/482#discussion_r278456353)